src/significance.jl

```diff
@@ -0,0 +1,19 @@
+function p_direction(x::Vector{Float64})
+    return maximum([sum(x .> 0) ./ length(x), sum(x .< 0) ./ length(x)])
+end
+
+function p_direction(chains::Chains; kwargs...)
+    # Store everything.
+    funs = [p_direction]
+    func_names = [:p_direction]
+
+    # Summarize.
+    summary_df = summarize(
+        chains, funs...;
+        func_names=func_names,
```

Comment:
To avoid alloctions:
```suggestion
    # Summarize.
    summary_df = summarize(
        chains, p_direction;
        func_names=[:p_direction],
```